### PR TITLE
Update Get-CsTeamsAcsFederationConfiguration.md

### PR DESCRIPTION
**Pull request by jamescadd**
Adds filter and identity parameters

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md [hunk 1]
```diff
@@ -56,6 +56,43 @@ EnableAcsUsers       : False
 
 ## PARAMETERS
 
+### -Filter
```
Comment (dstrome) [inline teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md:59]: Parameters have been added to the parameters list but not to the syntax block. Please update the syntax block.

File: teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md [hunk 2]
```diff
@@ -56,6 +56,43 @@ EnableAcsUsers       : False
...
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
```
Comment (dstrome) [inline teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md:76]: Same as with the filter parameter. Add to syntax block.

**Review by dstrome — CHANGES_REQUESTED** [review]
Please see comments in the article.

File: teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md (hunk 1 above)
Comment (jamescadd) [reply teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md:61]: Addressed

File: teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md (hunk 2 above)
Comment (jamescadd) [reply teams/teams-ps/teams/Get-CsTeamsAcsFederationConfiguration.md:78]: Addressed

**Review by jamescadd — COMMENTED** [review]
Syntax block now contains filter and identity

**Pull request merged**